### ISSUE TITLE
Unable to build from "master" branch

### ISSUE THREAD

**Issue by SchweinDeBurg** [fish-shell/fish-shell#6126]
```
[108/110] Building man pages with Sphinx
FAILED: CMakeFiles/sphinx-manpages 
cd /home/elijah/rpmbuild/BUILD/fish/3.14.5-x86_64-redhat-linux-gnu && /usr/bin/sphinx-build-3 -q -b man -c /home/elijah/rpmbuild/BUILD/fish/sphinx_doc_src -d /home/elijah/rpmbuild/BUILD/fish/3.14.5-x86_64-redhat-linux-gnu/user_doc/doctrees /home/elijah/rpmbuild/BUILD/fish/sphinx_doc_src /home/elijah/rpmbuild/BUILD/fish/3.14.5-x86_64-redhat-linux-gnu/user_doc/man/man1

Exception occurred:
  File "/usr/lib64/python3.6/subprocess.py", line 1344, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'fish_indent': 'fish_indent'
The full traceback has been saved in /home/elijah/.local/tmp/sphinx-err-ogn2gyx6.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
ninja: build stopped: subcommand failed.
error: Bad exit status from /home/elijah/.local/tmp/rpm-tmp.Jt7UKZ (%build)
```

Contents of 'sphinx-err-ogn2gyx6.log' follows:
```
# Sphinx version: 1.5.6
# Python version: 3.6.5 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.9.6
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/sphinx/cmdline.py", line 295, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/usr/lib/python3.6/site-packages/sphinx/application.py", line 163, in __init__
    confoverrides or {}, self.tags)
  File "/usr/lib/python3.6/site-packages/sphinx/config.py", line 134, in __init__
    execfile_(filename, config)
  File "/usr/lib/python3.6/site-packages/sphinx/util/pycompat.py", line 129, in execfile_
    exec_(code, _globals)
  File "conf.py", line 57, in <module>
  File "/usr/lib64/python3.6/subprocess.py", line 403, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib64/python3.6/subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.6/subprocess.py", line 1344, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'fish_indent': 'fish_indent'
```

**Comment by floam** [comment]
I think https://github.com/fish-shell/fish-shell/commit/3b8505bebeebd79c9cd3a3aa39a7404725db985a puts you in a catch-22 if `fish_indent` is not yet installed.

**Comment by SchweinDeBurg** [comment]
Many thanx for your suggestion! I've added CMake build directory to the $PATH environment variable in the %build and %install sections of .spec file and then fish rpm package was successfully built.

**Comment by zx8** [comment]
This may be the cause of an issue I reported on Gitter yesterday (https://gitter.im/fish-shell/fish-shell?at=5d83413ba38dae3a637c26d8)?

> **zx8 @zx8 Sep 19 09:50**
> I build `fish` from the latest `master` commit every morning. Something broke in the last 24 hours which is causing the build to fail on macOS:
```sh
[ 90%] Linking CXX static library libfishlib.a
/usr/local/Cellar/cmake/3.15.3/bin/cmake -P CMakeFiles/fishlib.dir/cmake_clean_target.cmake
/usr/local/Cellar/cmake/3.15.3/bin/cmake -E cmake_link_script CMakeFiles/fishlib.dir/link.txt --verbose=1
/usr/bin/ar qc libfishlib.a  <lots of cpp.o files>
/usr/bin/ranlib libfishlib.a
[ 91%] Built target fishlib
make: *** [all] Error 2
````

**Comment by zanchey** [comment]
That change should not be necessary; the build is supposed to insert the build path into `$PATH`. I'll take a look.

**Comment by zanchey** [comment]
Wait, I missed this for the manpages; I'll update the CMakeFile.

@zx8 that's a separate problem; you might consider opening another issue if the suggestions on Gitter did not work for you.

**Comment by zanchey** [comment]
Closed in 8e0aa03c4a315290c96c6d8afffbfdde89b66c49 where I fatfingered the issue number, again.

**Comment by zx8** [comment]
@zanchey Don't think it's a separate problem, my Homebrew install works again since your fix:

```
$ brew install fish --HEAD
==> Cloning https://github.com/fish-shell/fish-shell.git
Updating /Users/zx8/Library/Caches/Homebrew/fish--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at 8e0aa03c cmake: sphinx-manpages requires fish_indent for version number
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/fish/HEAD-8e0aa03 -DCMAKE_BUILD_TYPE=Release -DCMA
==> make install
🍺  /usr/local/Cellar/fish/HEAD-8e0aa03: 1,180 files, 12.4MB, built in 1 minute 6 seconds
```

Thanks!
